src/components/TodoList/TodoList.tsx

```diff
@@ -1,14 +1,47 @@
 /* eslint-disable */
 import React from 'react';
+import classNames from 'classnames';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { clearTodo, selectTodo } from '../../features/currentTodo';
+import { Todo } from '../../types';
 
 export const TodoList: React.FC = () => {
+  const todos = useAppSelector(state => state.todos)
+  const currentTodo = useAppSelector((store) => store.currentTodo);
+  const { query, status } = useAppSelector(state => state.filter);
+  
+  const dispatch = useAppDispatch();
+
+  const filteredTodo = todos.filter(todo => {
+    const matchesStatus = 
+    status === 'all' 
+      ? true 
+      : status === 'active'
+        ? !todo.completed
+        : todo.completed
+    const matchesQuery = todo.title.toLowerCase().includes(query.toLowerCase());
+
+    return matchesStatus && matchesQuery;
+  })
```

Comment:
The `status` comparison uses the string 'all', but it should match the `FilterTypes` enum values. Ensure that the `status` variable is compared against the correct enum values to avoid potential issues.